### DISCUSSION TITLE
.NETCore API Client

### DISCUSSION THREAD

**Discussion by Ebenderooock** [openfoodfacts/.github#26] [Ideas]
Are there any plans to create a simple .NET Api Client? Can I submit my implementation?

**Comment by teolemon** [comment]
We have an ongoing SDK project that would need support: https://github.com/openfoodfacts/openfoodfacts-csharp

We also have some older projects on taxonomy handling in c# by @hangy 

https://wiki.openfoodfacts.org/API/Scripts

**Comment by teolemon** [comment]
@Ebenderooock
